### PR TITLE
Remove `Post#to_param` again

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Comment by tricknotes** [comment]
We want to have some posts that have `public_id` different with `id`.
How do you think about updating seed data like the following?

``` diff
diff --git a/db/seeds/development/posts.seeds.rb b/db/seeds/development/posts.seeds.rb
index e800e39..f9579f0 100644
--- a/db/seeds/development/posts.seeds.rb
+++ b/db/seeds/development/posts.seeds.rb
@@ -8,6 +8,7 @@ after "development:sites", "development:categories" do

       This is first post for site 1!
     EOS
+    public_id: 10000, # some large number
     thumbnail: open(Rails.root.join("db/data/thumbnail.jpg")),
     published_at: Time.current,
     category: site1.categories.find_by!(slug: "category1"),
```

**Comment by okkez** [comment]
> How do you think about updating seed data?

I think we need another post(record) for this purpose.
Because sometimes `post.id == post#public_id`.

**Comment by tricknotes** [comment]
I think all posts have `public_id` that is difference with `id`.
Are there any reason why `Post#id` should be the same as `Post#public_id`?

**Comment by okkez** [comment]
In production:

```
d3oiv5f6lpg219=> select count(*) from posts;
 count 
-------
  5279
(1 行)

d3oiv5f6lpg219=> select count(*) from posts where id = public_id;
 count 
-------
   261
(1 行)

d3oiv5f6lpg219=> select count(*) from posts where id <> public_id;
 count 
-------
  5018
(1 行)
```

File: db/seeds/development/posts.seeds.rb [hunk 1]
```diff
@@ -49,4 +49,41 @@
...
+  site1.posts.create!(
+    title: "Hello",
+    body: <<-EOS.strip_heredoc,
+      # Hello
+      hi !
+
+      This is post `id != public_id`!
+    EOS
+    public_id: 100_000,
+    thumbnail: open(Rails.root.join("db/data/thumbnail.jpg")),
```
Comment (tricknotes) [inline db/seeds/development/posts.seeds.rb]: You can use `Pathname#open` to open a file (not a URL) explicitly.

File: db/seeds/development/posts.seeds.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
       This is first post for site 1!
     EOS
-    thumbnail: open(Rails.root.join("db/data/thumbnail.jpg")),
+    thumbnail: File.open(Rails.root.join("db/data/thumbnail.jpg")),
```
Comment (tricknotes) [inline db/seeds/development/posts.seeds.rb]:
``` ruby
Rails.root.join("db/data/thumbnail.jpg").open
```

**Comment by tricknotes** [comment]
Also this should be updated? https://github.com/bm-sms/daimon-news/blob/d740a3c5ec3f35f9121ed5c6b62ecc68fb9cbb33/config/breadcrumbs.rb#L19

**Comment by okkez** [comment]
YES, we need updating breadcrumb!

**Comment by tricknotes** [comment]
🚀

**Pull request merged**